airbyte-webapp/src/components/EntityTable/components/StreamStatusCell.tsx

```diff
@@ -0,0 +1,178 @@
+import { CellContext } from "@tanstack/react-table";
```

Comment:
EntityTable is a bit overloaded, and this component should be standalone moved to `components/connection/StreamStatusCell` so that it can be easily broken down into multiple files.

Reply:
Some of this logic is going to be shared with the stream-centric status page. I was planning on extracting that out then.